umbraco-workflow/upgrading-workflow.md

```diff
@@ -0,0 +1,74 @@
+# Upgrading Umbraco Workflow
+
+This article shows how to manually upgrade Umbraco Workflow to run the latest version. Umbraco Workflow displays a prompt in the **Workflow** section when a new version is available.
+
+{% hint style="info" %}
+If you are migrating from Plumber to Umbraco Workflow, see the [Migration from Plumber to Umbraco Workflow](#migration-from-plumber-to-umbraco-workflow) section.
+{% endhint %}
+
+## Get the latest version of Umbraco Workflow
+
+To get the latest version of Umbraco Workflow, you can upgrade using either of the two options:
+
+- [NuGet](#nuget)
+- [Visual Studio](#visual-studio)
+
+### NuGet
+
+NuGet installs the latest version of the package when you use the `dotnet add package Umbraco.Workflow` command unless you specify a package version:
+
+  `dotnet add package Umbraco.Workflow --version <VERSION>`
+
+After you have added a package reference to your project by executing the `dotnet add package Umbraco.Workflow` command in the directory that contains your project file, run `dotnet restore` to install the package.
+
+### Visual Studio
+
+- Go to `Tools` -> `NuGet Package Manager` -> `Manage NuGet Packages for Solution...` in Visual Studio, to upgrade Umbraco Workflow:
+- Select **Umbraco.Workflow**.
+- Select the latest version from the **Version** drop-down and click **Install**.
+
+  ![NuGet Package Manager](images/Manage_packages_v10.png)
+
+- Open the **<project-name>.csproj** file to make sure the package reference is updated:
+
+  ```xml
+  <ItemGroup>
+    <PackageReference Include="Umbraco.Workflow" Version="11.x.x" />
+  </ItemGroup>
+  ```
+
+## Migration from Plumber to Umbraco Workflow
+
+If you are upgrading an Umbraco 10 installation with an existing Plumber installation, follow these steps:
```

Comment:
Would it make sense to explain why you need to follow these steps when upgrading from Umbraco 10?
I don't think that's very clear at the moment. 
Is it because Plumber will not work on Umbraco 11?

Reply:
Yup, cz the namespaces have been updated from Plumber to Workflow. I have added that info 🙂